client/web/src/components/popover/UserPopover/GroupUserPopover.tsx

```diff
@@ -84,6 +95,12 @@ export const GroupUserPopover: React.FC<{
         <div className="pt-2">{pluginUserExtraInfoEl}</div>
 
         <div className="text-right">
+           <Tooltip title={t('添加好友')}>
```

Comment:
i think its should also have a permission check like `allowSendMessage` because its should can be close if some group dont wanna allow user DM